<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QWidget" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableInventaris">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kategori</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Nama Barang: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNamaBarang">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Katagori</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Jumlah</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKategori">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Alat Tulis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buku</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Perlengkapan</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="sbJumlah">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTambah">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Urutkan :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kategori</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnUrut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Urutkan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>401</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
